runtime/include/tasks/qthreads/tasks-qthreads.h

```diff
@@ -51,6 +51,9 @@ void chpl_task_yield(void);
 //
 typedef unsigned int chpl_taskID_t;
 #define chpl_nullTaskID QTHREAD_NULL_TASK_ID
+#ifndef CHPL_TASK_ID_STRING_MAX_LEN
+#define CHPL_TASK_ID_STRING_MAX_LEN 20
```

Comment:
For safety probably should be 21, appropriate for a 64-bit uint plus a trailing '\0'.  (There are C compilers, typically HPC-oriented, that can make the size of a default int 64 bits.)